pkgs/tools/misc/kisslicer/default.nix

```diff
@@ -0,0 +1,55 @@
+{ fetchurl
+, libX11
+, mesa
+, makeWrapper
+, stdenv
+, unzip
+, inidir ? "\\\$HOME/.config/kisslicer"
+}:
+
+let
+  name = "kisslicer";
+  version = "1.6.2";
+
+  libPath = stdenv.lib.makeLibraryPath [
+    mesa
+    stdenv.cc.cc
+    libX11
+  ];
+
+in stdenv.mkDerivation rec {
+  inherit version name;
+
+  src = fetchurl {
```

Comment:
I'd suggest to use
```
src = fetchzip {
  url = "http://www.kisslicer.com/uploads/1/5/3/8/15381852/kisslicer_linux64_1.6.2_release.zip";
  sha256 = "blahblahblah";
  stripRoot = false;
};
```
since then you can remove the custom `unpackPhase`.

Reply:
I've switched to it. I'm not a big fan of the behavior as I have to unpack it in order to get the SHA (though yes, there's `nix-prefetch-url -A`) but it drops the unpackphase.

Reply:
Hmm, I don't think you'll need to worry about the unpack phase since `fetchzip` extracts the archive after the fetch. Try removing the `phases = [ "unpackPhase" "installPhase" "fixupPhase" ];` line. It should still work fine.

Reply:
It does work without it, but is there value in skipping phases that are a noop? There isn't a configure script or Makefile so it cuts down a small amount of logging that isn't pertinent to the build.

Reply:
Generally speaking it is best not to override the phases field since it is controlling fairly low level stuff within stdenv and it is easy to miss something. The recommendation I've seen is to instead disable the phases that are not needed. In this case to have
```
  configurePhase = ":";
  dontBuild = true;
```
instead of `phases = [ … ];`. Don't ask me why there is no `dontConfigure`.

In any case, I believe the phases you have set is OK so I'll merge as is. Can also sort it out later if any problem ever arises.